configure.ac

```diff
@@ -4515,6 +4515,19 @@ else
 fi
 
 
+# MemUse Entropy
+AC_ARG_ENABLE([entropy-memuse],
+    [AS_HELP_STRING([--enable-entropy-memuse],[Enable memuse entropy support (default: disabled)])],
+    [ ENABLED_ENTROPY_MEMUSE=$enableval ],
+    [ ENABLED_ENTROPY_MEMUSE=no ]
+    )
+
```

Comment:
`-memuse` requires `-hashdrbg`. Should we fail early in `./configure ` process instead of build error with `./configure  --enable-entropy-memuse --disable-hashdrbg` && make
```
./wolfssl/wolfcrypt/random.h:117:6: error: #error No RNG source defined!
  117 |     #error No RNG source defined!
```

Reply:
Similarly, `-memuse `requires `-sha3`. Should we check if it's enabled or force enable it?

Reply:
Doesn't need hashdrbg.
MemUse Entropy is called in wc_GenerateSeed() which is a public API.

Reply:
Added support for: thread and nofallback

Reply:
SHA-3 is now enabled when MemUse Entropy is configured.